actions/websocket_actions.jsx

```diff
@@ -1408,9 +1410,25 @@ function handleCloudPaymentStatusUpdated() {
 function handleRefreshAppsBindings() {
     return (doDispatch, doGetState) => {
         const state = doGetState();
-        if (appsEnabled(state)) {
-            doDispatch(fetchAppBindings(getCurrentUserId(state), getCurrentChannelId(state)));
+        if (!appsEnabled(state)) {
+            return {data: true};
+        }
+
+        doDispatch(fetchAppBindings(getCurrentChannelId(state)));
+
+        const rhsPost = getSelectedPost(state);
+        const {params: {threadIdentifier}} = useRouteMatch();
```

Comment:
The React docs specifically recommend using hooks outside of a React context, but I don't know how else we'd do that since the websocket lives outside of React state. I'd be okay with keeping this as long as it works though.

Maybe we should double check that though since looking at the source code for `useRouteMatch`, it looks like it relies on React context which wouldn't be present here ([link](https://github.com/ReactTraining/react-router/blob/42933fe141819e4662113ab2c320bf86be3490fb/packages/react-router/modules/hooks.js#L44)).

Reply:
I have to admit I haven't checked whether this piece of code is working properly (not many ways to check from my side at the moment). Will figure out how to test it and confirm whether it works or not.

If it doesn't, I will change it to something like:
`const threadIdentifier = new URL(window.location.href).searchParams.get('threadIdentifier');`

Reply:
You were right, this was not working due to the hook being used outside of a component.
Also, my first solution was not working because the url is slightly different than what I thought from the other logic. Not sure what url is the other logic using.
A new version is in place.